### DISCUSSION TITLE
The SVG view should be zoomed in and out with the mouse position as the center

### DISCUSSION THREAD

**Discussion by wuyang26** [ElinamLLC/SharpVectors#293] [Ideas]
SVG views are based on center scaling, rather than scaling based on the current mouse position

**Comment by paulushub** [comment]
@wuyang26 The ZoomAndPanel is from the [CodeProject article](https://www.codeproject.com/Articles/85603/A-WPF-custom-control-for-zooming-and-panning), used for demo and added to the Runtime library for convenience.
When converted, the rendered image is basically in WPF domain, and Windows/WPF standard behavior are expected here.
For that matter, no SVG specific changes were made to that control. 
You are can submit SVG specific changes, preferably in an extended control to maintain the current behavior for those currently using it.
